lib/password.ex

```diff
@@ -27,4 +27,8 @@ defmodule Fields.Password do
   def load(value) do
     {:ok, value}
   end
+
+  def embed_as(_), do: :self
```

Comment:
We need to add this code to our encryption example see: https://github.com/dwyl/phoenix-ecto-encryption-example/issues/31